packages/fxa-content-server/docs/relier-communication-protocols/oauth-webchannel.md

```diff
@@ -0,0 +1,24 @@
+# Communication with OAuth WebChannels
+
+OAuth WebChannels is an extension of the [FxA WebChannel Desktop Flow](fx-webchannel.md).
+It consists of similar messages as the Desktop flow. Some behaviours in the OAuth flow are different.
+
+This flow currently supports the following messages:
+
+-   'FXA_STATUS'
```

Comment:
The semantics of `fxa-status` are subtly different IIUC, so it's probably worth talking about those differences here (e.g. with the behaviour of engines).